src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const transformedString = sourceString
+    .split(';')
+    .filter((style) => style.trim() !== '')
```

Comment:
```suggestion
    .filter((style) => style.trim())
```
You don't need to compare with an empty string, just check if exists